source/_components/climate.tfiac.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Tfiac AC"
+description: "Instructions on how to integrate Tfiac AC devices with Home Assistant."
+date: 2019-03-10 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tfiac.png
+ha_category: Climate
+ha_release: 0.88
+ha_iot_class: "Local Polling"
+redirect_from:
```

Comment:
Please remove this

Reply:
Not sure if you are suggesting to delete lines 13 and 14.

Reply:
Line 14 and 15